app/serializers/auction_serializer.rb

```diff
@@ -43,6 +43,15 @@ def winning_bid
   private
 
   def find_winning_bid
-    RulesFactory.new(object).create.winning_bid
+    if auction_status.available?
+      NullBid.new
+    else
+      RulesFactory.new(object).create.winning_bid
```

Comment:
Oh I'm not asking you to justify all your actions, sorry! I just grapple with this same question and was wondering when the threshold is reached.